src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -218,7 +218,7 @@ protected override async Task LoadComponentsAsync(CancellationToken cancellation
 
             // we need to load it as early as possible since we can have errors from
             // package from each language very early
-            await this.ComponentModel.GetService<TaskCenterSolutionAnalysisProgressReporter>().InitializeAsync(this).ConfigureAwait(false);
+            await this.ComponentModel.GetService<TaskCenterSolutionAnalysisProgressReporter>().InitializeAsync().ConfigureAwait(false);
```

Comment:
Why `InitializeAsync()` instead of `initializeAsync(this)`?

Reply:
➡️ `this` was only passed so the target would have access to a service provider. It now gets the service provider from MEF during construction.